### PR TITLE
feat(Icon): add an examples for color prop

### PR DESCRIPTION
**Pull request by thaytharma**
Added examples for color property in `Icon`.
Also added some info about using CSS variables as a color option in the docs.

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/components/icon/properties.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ showTabs: true
...
 | `size`                                      | _(optional)_ the dimension of the icon. This will be the `viewBox` and represent `width` and `height`. Defaults to `16`. You can use `small`,`medium`, `large` or `auto`. Auto will enable that the icon size gets inherited by the parent HTML element if it provides a `font-size`. |
-| `color`                                     | _(optional)_ sets a color property to the `svg` markup. Default is no color, which means _black_                                                                                                                                                                                       |
+| `color`                                     | _(optional)_ sets a color property to the `svg` markup. Default is no color, which means _black._ `color` can be any valid color property, such as Hex, RGB or any CSS variable from the [colors table](/uilib/usage/customisation/colors), e.g. `var(--color-ocean-green)`.                                                                                                                                  |
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/uilib/components/icon/properties.md:14]:
```suggestion
| `color`                                     | _(optional)_ the color` can be any valid color property, such as Hex, RGB or preferable – any CSS variable from the [colors table](/uilib/usage/customisation/colors), e.g. `var(--color-ocean-green)`. Default is no color, which means `--color-black-80`.                                                                                                                                  |
```

**Review by tujoworker — APPROVED** [review]
🙌 Awesome work! 🙌

I just have a suggestion on the property description.

File: packages/dnb-design-system-portal/src/docs/uilib/components/icon/properties.md (hunk 1 above)
Comment (thaytharma) [reply packages/dnb-design-system-portal/src/docs/uilib/components/icon/properties.md]: This has been updated 😊

**Pull request merged**